src/scripts/main.js

```diff
@@ -1,3 +1,381 @@
 'use strict';
 
-// write code here
+const tableHeaders = [...document.querySelectorAll('thead th')];
+let tableRows = [...document.querySelectorAll('tbody tr')];
+const tbody = document.querySelector('tbody');
+let orderSort = true;
+let lastClickedHeader = null;
+let indexRow = null;
+
+const form = document.createElement('form');
+
+form.classList.add('new-employee-form');
+
+const createInput = (type, value, dataQa) => {
+  const input = document.createElement('input');
+
+  input.setAttribute('type', type);
+  input.setAttribute('name', value);
+  input.setAttribute('data-qa', dataQa);
+  input.setAttribute('required', true);
+
+  return input;
+};
+
+const createLabel = (text) => {
+  const label = document.createElement('label');
+
+  label.textContent = text;
+
+  return label;
+};
+
+const createOption = (text, value) => {
+  const option = document.createElement('option');
+
+  option.textContent = text;
+  option.setAttribute('value', value);
+
+  return option;
+};
+
+const inputName = createInput('text', 'name', 'name');
+const inputPosition = createInput('text', 'position', 'position');
+const inputAge = createInput('number', 'age', 'age');
+const inputSalary = createInput('number', 'salary', 'salary');
+
+const button = document.createElement('button');
+
+button.textContent = 'Save to table';
+
+const labelName = createLabel('Name:');
+const labelPosition = createLabel('Position:');
+const labelCountry = createLabel('Country:');
+const labelAge = createLabel('Age:');
+const labelSalary = createLabel('Salary:');
+
+const selectCountry = document.createElement('select');
+
+form.setAttribute('id', 'formEmployees');
+selectCountry.setAttribute('name', 'office');
+selectCountry.setAttribute('data-qa', 'office');
+button.setAttribute('type', 'submit');
+button.setAttribute('form', 'formEmployees');
+
+const countries = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
```

Comment:
The task requires the select element to have 6 options, including 'San Francisco'. Ensure that all required options are included in the `countries` array.